src/RecoveryServices/RecoveryServices.SiteRecovery/ReplicationProtectedItem/NewAzureRmRecoveryServicesAsrReplicationProtectedItem.cs

```diff
@@ -611,6 +613,11 @@ private void AzureToAzureReplication(EnableProtectionInput input)
                 RecoveryBootDiagStorageAccountId = this.RecoveryBootDiagStorageAccountId
             };
 
+            if (!string.IsNullOrEmpty(this.ReplicationGroupName))
```

Comment:
Use PSCmdlet.IsParameterBound pattern, and protect this parameter with a ValidateNotNullOrEmpty attribute